### PR TITLE
chromium: update to 70.0.3538.67.

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by Cogitri** [comment]
Currently building...

File: srcpkgs/chromium/template [hunk 1]
```diff
@@ -160,6 +159,9 @@ do_configure() {
 		enable_widevine=true
 		enable_hangout_services_extension=true'
 
+	#  Jumbo/Unity builds: https://chromium.googlesource.com/chromium/src/+/master/docs/jumbo.md
+	conf+="use_jumbo_build=true"
+
```
Comment (Cogitri) [inline srcpkgs/chromium/template]: Not quite sure how beefy the builders are, jumbo builds can take up to 3-4GB per build job for linking. However, it's _way_ faster than building without jumbo building (jumbo building chromium takes about 1 1/2 hours vs. 6-8 hours without it on my Ryzen 1700) and upstream only tests with jumbo building - although building without jumbo building usually doesn't need lots of patches, in this case only https://github.com/void-linux/void-packages/pull/3880/files#diff-12a5c377aace0348303db474b2e9bafa

Comment (maxice8) [reply srcpkgs/chromium/template]: @Duncaen

**Comment by Cogitri** [comment]
Tested on

- [X] x86_64
- [ ] x86_64-musl
- [ ] i686

File: srcpkgs/chromium/template (hunk 1 above)
Comment (Cogitri) [reply srcpkgs/chromium/template]: I've optionalised it for now.

**Comment by Cogitri** [comment]
Also, other distros start including downstream patches to enable VAAPI decoding for chromium, which lowers CPU usage significantly. Not quite sure if we should also include something like that. Here's the patch Fedora uses: https://github.com/biswasab/chromium-vaapi/blob/master/vaapi.patch

**Comment by maxice8** [comment]
Yes please for VAAPI patch

@void-linux/pkg-committers

File: srcpkgs/chromium/template (hunk 1 above)
Comment (Vaelatern) [reply srcpkgs/chromium/template]: If it's just 3-4 GB for builds, and not, say, 10, I think the relevant builders will do fine with jumbo builds.

Comment (Cogitri) [reply srcpkgs/chromium/template]: Alright, I'll enable it by default, I guess we'll see if that works out :)

**Comment by Duncaen** [comment]
I don't think `INSTALL.msg` is the right place for something like this, its kinda long and people already ignore most of the messages, adding more optional information doesn't improve the situation.

**Comment by Cogitri** [comment]
Hm, what would you do then? Sadly we don't have optional deps for this

**Comment by Vaelatern** [comment]
How about the wiki for that documentation. But I agree, INSTALL.msg is a very bad place to put most warnings.

**Comment by Cogitri** [comment]
I feel like even less people (no one) will look at the wiki for this - simply because most people won't have the necessary VAAPI driver installed, so they won't notice that anything has changed

**Comment by Cogitri** [comment]
Another thing: We could stop building chromium_sandbox, which would spare us from having a setuid chrome binary, but requires USER_NS to be enabled in the kernel - but it seems like we have that enabled anyway.

**Comment by Cogitri** [comment]
Needs #1475 for VAAPI

Builds fine on x86_64, segfaults on x86_64-musl with clang and doesn't link with GCC, let's see...

**Comment by Cogitri** [comment]
Alright, works on x86_64{,-musl} now, currently compiling for i686...

**Comment by Cogitri** [comment]
Seems like not building with jumbo building isn't an option for musl BTW, it fails to link on musl without jumbo building.

**Comment by Cogitri** [comment]
Builds on all platforms now; please test.

**Comment by maxice8** [comment]
libva2 was merged

**Comment by Gottox** [comment]
lgtm :+1:

**Pull request merged**